OrnithologistsGuild/Content/ContentManager.cs

```diff
@@ -29,7 +32,87 @@ private static void Content_AssetRequested(object sender, AssetRequestedEventArg
             {
                 e.LoadFromModFile<Texture2D>("assets/binoculars.png", AssetLoadPriority.Low);
             }
+            // mail & triggers
+            // this can be done in content patcher ofc, kept here for i18n key.
+            if (e.Name.IsEquivalentTo("Data/Mail"))
+            {
+                e.Edit(Edit_DataMail);
+            }
+            if (e.Name.IsEquivalentTo("Data/TriggerActions"))
+            {
+                e.Edit(Edit_DataTriggerActions);
+            }
         }
+
+        private static void Edit_DataMail(IAssetData data)
+        {
+            var mail = data.AsDictionary<string, string>().Data;
+            // give lifelist
+            mail["Mods_Ivy_OrnithologistsGuild_Introduction"] = $"{I18n.Mail_Introduction()}%item id (T)Ivy_OrnithologistsGuild_LifeList %% %item conversationTopic Ivy_OrnithologistGuild_Introduction 14 %% [#]{I18n.Mail_Introduction_Title()}";
```

Comment:
I think there are constants for items like `(T)Ivy_OrnithologistsGuild_LifeList` somewhere...